index.html

```diff
@@ -398,13 +388,13 @@ <h2>Stakeholders and Roles</h2>
 	  document for full definitions:
 	</p>
 	<ul>
-          <li id="STAKEHOLDER-Device-Manufacturer">Device Manufacturer</li>
-          <li id="STAKEHOLDER-System-Provider">System Provider</li>
-          <li id="STAKEHOLDER-System-Integrator">System Integrator</li>
-          <li id="STAKEHOLDER-System-Installer">System Installer</li>
-          <li id="STAKEHOLDER-System-User">System User</li>
-          <li id="STAKEHOLDER-System-Owner">System Owner</li>
-          <li id="STAKEHOLDER-System-Maintainer">System Maintainer</li>
+          <li id="STAKEHOLDER-Device-Manufacturer">Device Manufacturer: Device makers are individuals, companies, or organizations that design and develop networked or IoT devices to be sold to consumers.</li>
+          <li id="STAKEHOLDER-System-Provider">System Provider: A system provider is an individual, company, or organization that provides systems, such as network systems and IoT systems, to other individuals or organizations.</li>

```

Comment:
It seems odd for the first of these STAKEHOLDERs to have a plural description, while all the others have single descriptions. Based on numeric superiority, I'm suggesting changing the first to match the rest.
```suggestion
          <li id="STAKEHOLDER-Device-Manufacturer">Device Manufacturer: A device maker is an individual, company, or organization that designs and develops networked or IoT devices to be sold to consumers.</li>
          <li id="STAKEHOLDER-System-Provider">System Provider: A system provider is an individual, company, or organization that provides systems, such as network systems and IoT systems, to other individuals or organizations.</li>

```